### PR TITLE
add example

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/OfficeDocs-SharePoint-PowerShell/issues/312

### CODE REVIEW

**Comment by yogkumgit** [comment]
@dariomws @tseward

**Review by dariomws — APPROVED** [review]

File: sharepoint/sharepoint-ps/sharepoint-online/Remove-SPOTenantCdnOrigin.md [hunk 1]
```diff
@@ -28,22 +28,30 @@ Removes a new origin from the Public or Private content delivery network (CDN).
...
 
 The example removes a CDN from a site level.
 
+### EXAMPLE 3
+
+```powershell
+Remove-SPOTenantCdnOrigin -CdnType Private -OriginUrl /sites/branding/Assets
+```
+
+This example removes a recently-removed organizational assets library from CDN.
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Remove-SPOTenantCdnOrigin.md:53]:
```suggestion
This example removes a recently-removed organizational assets library from a CDN.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by tseward — APPROVED** [review]

**Comment by yogkumgit** [comment]
@get-itips

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**